src/java.base/share/classes/java/security/cert/PKIXParameters.java

```diff
@@ -333,6 +335,11 @@ public List<CertStore> getCertStores() {
      * revocation checking mechanism is to be substituted (by also calling the
      * {@link #addCertPathChecker addCertPathChecker} or {@link
      * #setCertPathCheckers setCertPathCheckers} methods).
+     * <p>
+     * However, if a {@code PKIXRevocationChecker} is passed in as a parameter
```

Comment:
The word "However" may be not necessary as the previous paragraph is ending with a substitute mechanism.  This sentence could be a further explanation of the  substitute mechanism.

Reply:
Yes, that's a fair point. I am going to change "However, if" to "Note that when". The reason is that I want to use some words to call special attention to this paragraph as the most important point of this is "it will be used to check revocation irrespective of the setting of the RevocationEnabled flag.", which the previous paragraphs did not explain.